audio-worklet/src/bitcrusher-demo.js

```diff
@@ -85,16 +113,50 @@ class BitcrusherDemo {
           default: this.masterGain_.gain.value,
           name: 'Volume'
         });
+
+    // GUI controls allow switching between audio worklet and script processor.
+    // The audio worklet node is used by default.
+    this.useAudioWorkletCheckBox_ = document.createElement('input');
+    this.useAudioWorkletCheckBox_.type = 'checkbox';
+    this.useAudioWorkletCheckBox_.checked = true;
+    const audioworkletLabel = document.createElement('label');
+    audioworkletLabel.textContent =
+        'Use Audio Worklet (if unchecked, use Script Processor Bitcrusher).';
+
+    this.useAudioWorkletCheckBox_.addEventListener(
+        'change', this.switchBitcrusherNode.bind(this));
+
+    let container = document.getElementById(containerId);
+    container.appendChild(this.useAudioWorkletCheckBox_);
+    container.appendChild(audioworkletLabel);
+  }
+
+  /**
+   * Switch between whether the audio worklet node or script processor node
+   * connects to the master gain node.
+   */
+  switchBitcrusherNode() {
+    if (this.useAudioWorkletCheckBox_.checked) {
+       this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
+       this.bitcrusherAudioWorklet_.disconnect();
+    } else {
+       this.bitcrusherAudioWorklet_.connect(this.masterGain_);
+       this.bitcrusherScriptProcessor_.output.disconnect();
+     }
   }
 
   /**
    * Change bit depth.
-   * This bound to event listener by a ParamController.
+   * This is bound to an event listener by a ParamController.
    * @param {Number} value the new bit depth
    */
   setBitDepth(value) {
     if (value < 1) console.error('The minimum bit depth rate is 1.');
-    this.bitcrusher_.bitDepth = value;
+    
+    this.bitDepthOffsetNode_.offset.value =
+          value - this.bitDepthDefaultValue_;
```

Comment:
I don't think this is relevant anymore since ConstantSourceNode will be removed.